sagemaker-fundamentals/create-training-job/create_training_job_encrypt_static_assets.ipynb

```diff
@@ -0,0 +1,1227 @@
+{
```

Comment:
<p>URL should be <code>https://raw.githubusercontent.com/aws/amazon-sagemaker-examples/master/sagemaker-fundamentals/execution-role/iam_helpers.py</code> ﻿</p><p>Is it worth checking to see if the user already has cloned the repository and has iam_helpers.py locally? I already had it in <code>../execution-role/</code>  from the original repository.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>right, but the <code>iam_helpers.py</code>  might not always be up to date, it's easier to maintain those helper functions if users always get the latest ones from the upstream </p>